client/src/features/project/components/AddCloudStorageButton.tsx

```diff
@@ -675,55 +653,34 @@ function SimpleAddCloudStorage({
               </FormText>
             </div>
 
-            {!CLOUD_STORAGE_READWRITE_ENABLED ? null : (
-              <div>
-                <div className="form-label">Mode</div>
-                <Controller
-                  control={control}
-                  name="readonly"
-                  render={({ field }) => (
-                    <>
-                      <div className="form-check">
-                        <Input
-                          type="radio"
-                          className="form-check-input"
-                          name="addCloudStorageReadOnlyRadio"
-                          id="addCloudStorageReadOnly"
-                          autoComplete="off"
-                          checked={field.value}
-                          onBlur={field.onBlur}
-                          onChange={() => field.onChange(true)}
-                        />
-                        <Label
-                          className={cx("form-check-label", "ms-2")}
-                          for="addCloudStorageReadOnly"
-                        >
-                          Read-only
-                        </Label>
-                      </div>
-                      <div className="form-check">
-                        <Input
-                          type="radio"
-                          className="form-check-input"
-                          name="addCloudStorageReadOnlyRadio"
-                          id="addCloudStorageReadWrite"
-                          autoComplete="off"
-                          checked={!field.value}
-                          onBlur={field.onBlur}
-                          onChange={() => field.onChange(false)}
-                        />
-                        <Label
-                          className={cx("form-check-label", "ms-2")}
-                          for="addCloudStorageReadWrite"
-                        >
-                          Read/Write
-                        </Label>
-                      </div>
-                    </>
-                  )}
-                />
-              </div>
-            )}
+            <div className="mb-3">
```

Comment:
Because this is the last item:

```suggestion
            <div>
```